pshtt/pshtt.py

```diff
@@ -53,14 +50,24 @@
     "Domain Enforces HTTPS", "Domain Uses Strong HSTS", "Unknown Error",
 ]
 
+# Used for caching the HSTS preload list from Chromium's source.
 PRELOAD_CACHE = None
+PRELOAD_CACHE_DEFAULT = "preloaded.json"
 preload_list = None
+
+# Used for caching the HSTS pending preload list from hstspreload.org.
+PRELOAD_PENDING_CACHE = None
+PRELOAD_PENDING_CACHE_DEFAULT = "preload-pending.json"
 preload_pending = None
 
 # Used for determining base domain via Mozilla's public suffix list.
-SUFFIX_CACHE = None
+PUBLIC_SUFFIX_CACHE = None
+PUBLIC_SUFFIX_CACHE_DEFAULT = "public-suffix-list.txt"
```

Comment:
I didn't provide a way to override the individual filenames within the third party cache directory. Though options are always nice, it doesn't seem like it has enough utility to merit the complexity.